ui/app/components/project-config-variables/list-item.hbs

```diff
@@ -0,0 +1,149 @@
+<li data-test-config-variables-list-item>
+  {{#if (or this.isEditing this.isCreating)}}
+    <form class="pds-form" data-test-config-variables-form {{on "submit" this.saveVariable}}>
+      <div part="fields" class="card">
+        <div class="card-header">
+          <h4>{{t "form.config_variables_settings.title"}}</h4>
+        </div>
+        <fieldset class="pds-formFieldSet">
+          <div>
+            <div class="pds-formField">
+              <label for="config-variable-name" class="pds-fieldName">
+                {{t "form.config_variables_settings.variable_name"}}
+              </label>
+              <Pds::Input
+                @type="text"
+                data-test-config-variables-var-name
+                id="config-variable-name"
+                placeholder={{t "form.config_variables_settings.variable_name_placeholder"}}
+                value={{this.variable.name}}
+                {{on "input" (pick "target.value" (set this "variable.name"))}}
+              />
+            </div>
+            <div class="pds-formField">
+              <label for="config-variable-value" class="pds-fieldName">
+                {{t "form.config_variables_settings.variable_value"}}
+              </label>
+              <div class="pds-formField__input-inline-toggle">
```

Comment:
no, the pds styles did not support the toggle layouts, so I extended them in https://github.com/hashicorp/waypoint/blob/main/ui/app/styles/components/structure/pds-form.scss#L60

Reply:
Cool, that makes sense. It looks like this field contains a text inputs though, so maybe we don’t need the toggle layout here, but on the field below?

Reply:
well we kind of need the selector here because it uses flexbox to align the child elements. This was done in https://github.com/hashicorp/waypoint/pull/1658 three months ago, so if we want to improve it, we could do so in another PR

Reply:
But unless I’m mistaken there’s only one child element in this case? For input variables, there’s an HCL toggle but I don’t see one here.

Reply:
oooh excuse me I was confused and thinking about the field that uses XToggle. You're right, I'm updating and changing that class